simulation_ws/src/cloudwatch_simulation/package.xml

```diff
@@ -17,8 +17,8 @@
   <depend>tf</depend>
   <depend>gazebo_ros</depend>
   <depend>gazebo_plugins</depend>
-  <!-- required to install .rviz model-->
-  <depend>turtlebot3_description</depend>
```

Comment:
This is not the same -- turtlebot3_description is still required, yes?  It provides the URDF, the robot 3D model.  

Separately, why does *simulation* require turtlebot3_navigation stack?  Maybe this is a legacy bug?

Reply:
We can keep `turtlebot3_model.rviz` instead of `turtlebot3_navigation.rviz` to avoid having `turtlebot3_navigation` as a dependency in simulation workspace

Reply:
I thought we always want to install `turtlebot3_navigation.rviz` when navigation is available, e.g. 
https://github.com/aws-robotics/aws-robomaker-bookstore-world/blob/ros1/CMakeLists.txt#L16
https://github.com/aws-robotics/aws-robomaker-small-house-world/blob/ros1/CMakeLists.txt#L17

Reply:
We can keep `turtlebot3_description` at this moment to unblock this PR.